behave/_types.py

```diff
@@ -55,8 +55,15 @@ def has_traceback(cls, exception):
     @classmethod
     def describe(cls, exception, use_traceback=False, prefix=""):
         # -- NORMAL CASE:
-        text = u"{prefix}{0}: {1}\n".format(exception.__class__.__name__,
-                                            exception, prefix=prefix)
+        try:
+            text = u"{prefix}{0}: {1}\n".format(exception.__class__.__name__,
+                                                exception, prefix=prefix)
+        except UnicodeDecodeError:
+            # Avoid UnicodeDecodeError in Python 2 when 'exception' is a system
+            # error thrown by a Windows host configured in a non-ascii language
+            exc_message = str(exception).decode('cp1252', 'ignore')
```

Comment:
That is a Windows specific solution that you applied to all platforms. This may bite other users on other platforms or with other Windows encoding / charset settings.

Reply:
This is of course a Windows specific solution, but it does not affect to other users, as UnicodeDecodeError is not handled in behave base code and they would only get an exception in any case.
Anyway, I updated it not to mask the original exception for non-Windows users.